pallets/liquidity-pools-gateway/Cargo.toml

```diff
@@ -12,6 +12,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0", features = ["derive"], default-features = false }
+hex = {version = "0.4.3", default-features = false }
```

Comment:
**CR**: We are importing `hex` without `std` here but not enabling the `std` feature below.